From 6f6c00a7b67bd0f45cc8631618b9e9606b0bace6 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Wed, 22 Jul 2026 09:31:32 +0100
Subject: [PATCH] misc: rp1-pio: Reverse christmas tree declarations

Linux devs prefer local variable declarations to be in order of
decreasing length, known as "reverse christmas tree" and it is easier
on the eye. Sort the declarations in rp1_pio_sm_config_xfer_internal
accordingly.

No functional change.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 drivers/misc/rp1-pio.c | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

--- a/drivers/misc/rp1-pio.c
+++ b/drivers/misc/rp1-pio.c
@@ -611,14 +611,13 @@ static int rp1_pio_sm_config_xfer_intern
 	struct rp1_pio_sm_set_dmactrl_args set_dmactrl_args;
 	struct rp1_pio_device *pio = client->pio;
 	struct platform_device *pdev = pio->pdev;
-	struct device *dev = &pdev->dev;
 	struct dma_slave_config config = {};
+	struct device *dev = &pdev->dev;
 	struct dma_slave_caps dma_caps;
+	struct dma_info *dma = NULL; 
 	phys_addr_t fifo_addr;
-	struct dma_info *dma = NULL;
 	uint32_t dma_mask;
 	char chan_name[4];
-	bool reconfigure = false;
 	int ret = 0;
 
 	if (sm >= RP1_PIO_SMS_COUNT || dir >= RP1_PIO_DIR_COUNT)
